tests/components/ct-vue-vite/playwright.config.ts

```diff
@@ -23,8 +23,8 @@ const config: PlaywrightTestConfig = {
   reporter: 'html',
   use: {
     ctTemplateDir: 'playwright',
-    ctCacheDir: 'playwright/.cache',
     trace: 'on-first-retry',
+    baseURL: 'http://localhost:3100'
```

Comment:
Otherwise `await expect(page).toHaveURL('/');` will throw an error with something along the lines of: URL is `http://localhost:3100/` instead of `/`. 

Maybe it's better to set a default `baseURL` in the `vitePlugin.ts`? Can i do that in a follow-up PR?